samples/graph-chat-lifecycle/nodejs/Helpers/ChatHelper.js

```diff
@@ -0,0 +1,196 @@
+const fs = require('fs');
```

Comment:
Folder folder and classes in node.js, please follow [camel casing](https://en.wikipedia.org/wiki/Camel_case).

Reply:
Changed all folders and class names following camel casing.